src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocNoPeriod.java

```diff
@@ -14,4 +14,9 @@ void foo4() throws Exception {}
 
     /** An especially short bit of Javadoc */
     void foo5() {}
+
+    /**
+     * {@summary No Period in end}
```

Comment:
`violation No Period in end`

Reply:
Sir, It will not throw any error as PERIOD as been set to `""` so no error here.

Reply:
Confirmed
https://github.com/checkstyle/checkstyle/blob/1e3fb3fae940bf7ea340592f7f0cda3af1b320e3/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheckTest.java#L99